app/routes/products.$categorySlug/route.tsx

```diff
@@ -140,7 +140,9 @@ export default function ProductsPage() {
 
                 <div className={styles.main}>
                     <div className={styles.categoryHeader}>
-                        <h1 className={styles.categoryName}>{category.name}</h1>
+                        <h1 className={styles.categoryName}>
+                            {appliedFilters.search ? `"${appliedFilters.search}"` : category.name}
```

Comment:
I don't think the category description from "All products" should be shown if we have a search term, as it's unclear what this description relates to:

<img width="968" alt="image" src="https://github.com/user-attachments/assets/450f3142-0a48-4567-826c-2b503ecae371">

Reply:
I think we also have a UX issue here. We show the search term in three places
1. The search box
2. The heading
3. The list of allied filters

Pressing "x" in the filter list clears the search term, but pressing "x" in the search box doesn't, which feels weird. So I think we either shouldn't persist the search term in the input, or "x" should do the same action in both cases.

@Dana-Levy

Reply:
> I don't think the category description from "All products" should be shown if we have a search term, as it's unclear what this description relates to

removed description.